web-client/public/js/warnings.js

```diff
@@ -0,0 +1,98 @@
+export var displayWarnings = function (warnings) {
```

Comment:
Another example of when to break out into a module: after studying this, I concluded that it was _completely_ self-contained. So it was a clear candidate for separation.

However, it doesn't have to be the only occupant in this file. You might find additional code relating to warnings that are cleanly separable. Now, you have a separate, clearly-named place to put that code.